docs/source/author/topics.rst

```diff
@@ -832,7 +837,27 @@ Programming Challenges Configuration File
         challenge belongs to.
 
       - ``challenge-number:`` The number order for this programming challenge.
-        Challenges are sorted this number.
+
+        .. note ::
```

Comment:
this note should probably be put at the end of the section for required and optional fields, for readability (maybe could add "see note below" next to the short description?